### ISSUE TITLE
Dashboard discussions

### ISSUE THREAD

**Issue by gondzo** [appirio-tech/connect-app#1934]
Topics/posts aren't shown on project dashboard (screens 5-8)

**Comment by maxceem** [comment]
What about bot notification? Looks like there is no mobile design for it, but I can adjust it on my taste if you wish.

![image](https://user-images.githubusercontent.com/146016/38170016-1e5d1764-35ad-11e8-841f-a72219df34d9.png)

**Comment by gondzo** [comment]
Yes please, let's show the notifications box as well

**Comment by maxceem** [comment]
As I understand **NEW STATUS** screen is supposed to create a new topic. But creating a new topic requires from us to define text and **title**. There is no field to define a title though. Could you plaese advise on this?

![image](https://user-images.githubusercontent.com/146016/38171614-6e0ab92a-35cf-11e8-9d69-399d74fb2d20.png)

**Comment by gondzo** [comment]
I think it was meant to be a 2 step process where you enter the title in the new status screen and enter text in the write comment screen. If so, the "Post Status" button name doesn't make sense, so let's rename it to "Next" or "Add comment"

@vic-topcoder @RishiRajSahu does this make sense?

**Comment by RishiRajSahu** [comment]
yeah make sense that's what I planned. Agree we need to rename the text on button, "Next" Sound as better option for posting comment during posting a status.

**Comment by RishiRajSahu** [comment]
are the changes done ?

**Comment by maxceem** [comment]
@RishiRajSahu almost, I think will finish in 1 hour.

**Comment by maxceem** [comment]
@RishiRajSahu done with this one

**Comment by maxceem** [comment]
3. Found issue with writing comments on a real device Android 7.0 browser Chrome version 65.0.3325.109
    **To reproduce**:
    - open any project
    - enter title of new topic - is ok
    - start enter text of the topic: once `space` or `Enter` is pressed, previously entered text dissapears.
   **Note**, this issue is not connected with the changes in this issue. It can be reproduced using mobile device on desktop version of the site also

**Comment by maxceem** [comment]
@gondzo What do you about replacing DraftJs editor with pure `textarea` on mobile devices? As we already don't have any styling buttons there I believe we will only lose user mention autocomplete.

**Comment by RishiRajSahu** [comment]
@vic-topcoder @gondzo what do you guys suggest ?

**Comment by gondzo** [comment]
using text area only would break user mentions completely, not just the autocomplete. We're using html format for the comments and mentions are links with title=@handle, for example `<a href="/users/TonyJ" title="@TonyJ">@Tony Jefts</a>`

Unless it's really complex to support draftJs on mobile, we should keep using it IMO

**Comment by gondzo** [comment]
oh, I didn't see all the comment updates @maxceem posted. Looking for alternatives..

**Comment by gondzo** [comment]
Looks like a simple textarea is the only viable option, draftjs won't be ready for mobile any time soon. @maxceem let's show the textarea instead of draftjs, and I'll create a separate issue to update tc-notifications to look for user mentions in plain text.

@vic-topcoder fyi this would mean we won't be able to support post editing nor any post formatting in mobile

**Comment by maxceem** [comment]
@gondzo @RishiRajSahu I've replaced DraftJS with textarea for mobile devices.

**Comment by maxceem** [comment]
@gondzo could you please clarify if I should implement things from the **vic-topcoder** comment:
1. replace `comment` with `post`
2. adjust the design of the system notifications (not sure how to trigger them)

**Comment by gondzo** [comment]
@maxceem yes please. 
system notifications are just regular notifications and are created on project status update, adding a link, adding a member, etc. We're talking about this block here 
![image](https://user-images.githubusercontent.com/3051646/38485612-bae8160a-3bda-11e8-8b13-ac2961105cc7.png) ?

**Comment by gondzo** [comment]
I think those are fine. 
agree @RishiRajSahu @vic-topcoder ?

**Comment by vic-topcoder** [comment]
seems fine to me. Thanks!
